src/MSBuild.Abstractions/NugetHelpers.cs

```diff
@@ -1,8 +1,10 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
 using System.Text.Json;
 using System.Threading.Tasks;
+using Microsoft.VisualBasic;
```

Comment:
Why is VB needed?

Reply:
good question, I think this is unnecessay but I'll check